.gitmodules

```diff
@@ -72,4 +72,7 @@
 	path = plugins/Morpheus/icons
 	url = https://github.com/matomo-org/matomo-icons.git
 
-# Note: do not add new plugin submodules here, but a few lines above
\ No newline at end of file
+# Note: do not add new plugin submodules here, but a few lines above
```

Comment:
any reason you did't add that in the list above (as the comment suggests)? 🤔

Reply:
moved it up, git adds it by default at the end